DEVELOPING.md

```diff
@@ -1,6 +1,6 @@
 ## Pre-requisites
 
-1.  We are using Node 8. If you need to work with multiple versions of Node, you
+1.  We are using Node 12. If you need to work with multiple versions of Node, you
```

Comment:
CLI isn't using 12 yet. Does this have a dep on 12 or does 10 still work?

Reply:
10 still works.  I think I was just being optimistic that we'd bundle and recommend 12, but not yet.  I'll change to 10 LTS.